apps/create-vesting/src/components/RecipientInput.tsx

```diff
@@ -1,13 +1,57 @@
 import { Input } from "@bleu/cow-hooks-ui";
+import { useState } from "react";
+
+export const RecipientInput = ({ value }: { value: string }) => {
+  const [displayValue, setDisplayValue] = useState(value);
+  const [isOnFocus, setIsOnFocus] = useState(false);
```

Comment:
Can you explain to me what is the behavior you are trying to implement here? I am wondering if there is a simpler way to do it